lib/eslint/eslint.js

```diff
@@ -59,6 +59,7 @@ const { version } = require("../../package.json");
  * @property {string} [resolvePluginsRelativeTo] The folder where plugins should be resolved from, defaulting to the CWD.
  * @property {string[]} [rulePaths] An array of directories to load custom rules from.
  * @property {boolean} [useEslintrc] False disables looking for .eslintrc.* files.
+ * @property {number} exitOnFatalError Number of fatal errors for the result.
```

Comment:
```suggestion
```

This is only a CLI option, we're not passing it to the ESLint class?

Reply:
To expand on my comment, this is JSDoc for [ESLint class](https://eslint.org/docs/developer-guide/nodejs-api#-new-eslintoptions). API options are very similar to the command-line options, but not the same. In this case, `--exit-on-fatal-errors` exists only on the command line. There is no `exitOnFatalError` API option, so that's why this JSDoc property definition should be removed.

Reply:
This is a good point, though — should this functionality exist in the ESLint class?

Reply:
> should this functionality exist in the ESLint class?

I think not, since ESLint class shouldn't control the process exit code.

A related functionality in the ESLint class could be to break early on the first occurrence of a parsing error, but I think that wasn't planned by the RFC.